modules/nw-multus-add-pod.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+
+[id="nw-multus-add-pod_{context}"]
+= Adding a Pod to an additional network
+
+You can add a Pod to an additional network. {product-title} will continue using
+the primary network for normal cluster related network traffic.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster.
+
+.Procedure
+
+To add a Pod to an additional network, complete the following steps:
+
+. Edit the Pod resource definition. If you are editing an existing Pod, run the
+following command to edit its definition in the default editor. Replace `<name>`
+with the name of the Pod to edit.
++
+----
+$ oc edit pod <name>
+----
+
+. In the Pod resource definition, include the `k8s.v1.cni.cncf.io/networks`
+parameter in the Pod `metadata` mapping. The `k8s.v1.cni.cncf.io/networks`
+accepts a comma separated string of one or more `AdditionalNetworkDefinition`
+names:
++
+[source,yaml] 
+----
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: <network>[,<network>,...] <1>
+----
+<1> Replace `<network>` with the name of the additional network to associate
+with the Pod. To specify more than one additional network, separate each network
+with a comma. Do not include any whitespace between the comma. If you specify
+the same additional network multiple times, the Pod will receive multiple IP
+addresses on that network.
++
+In the following example, two additional networks are assigned to the Pod:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example-pod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: vlan,vlan
+spec:
+  containers:
+  - name: example-pod
+    command: ["/bin/bash", "-c", "sleep 2000000000000"]
+    image: centos/tools
+----
+
+. Optional: Confirm that the annotation exists in the Pod CR by running the
+following command:
++
+----
+$ oc get pod <name> -o go-template='{{.metadata.annotations}}'
+----
+////
+What's a proper way to get this annotation?
```

Comment:
My typical way is to actually just do a wholesale `oc describe pod <name>` and just look at the annotations section.

However, to surgically pick it up, what you have is perfectly acceptable.

You can also use the JSONpath, too, like:

```
oc get pod <name> -o=jsonpath='{.metadata.annotations}'
```

Reply:
@dougbtv, I've been experimenting with getting information out of OCP. In this case, I was probably being overly ambitious. `describe` may be the most intuitive way. Someone that wants to extract specific information may know `oc` well enough to craft a better command than I did.